src/questions/blockchain_basics.yaml

```diff
@@ -1,35 +1,1100 @@
+  answerKeys:
+    - A
+    - C
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Very less transaction Fess
+      key: A
+      order: 0
+    - content: Extremely volatile nature
+      key: B
+      order: 1
+    - content: Strong backing asset
+      key: C
+      order: 2
+    - content: High tangible value
+      key: D
+      order: 3
+- uuid: c12ae1b5-3940-496a-88fa-41be29142f44
+  type: SingleChoice
+  content: Where do you keep your cryptocurrency private key?
+  hint: NoHint
+  explanation: Cryptocurrency private key must be kept away from the internet.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Wallet
+      key: A
+      order: 0
+    - content: Bank Account 
+      key: B
+      order: 1
+    - content: Floppy disk
+      key: C
+      order: 2
+    - content: Drive Storages
+      key: D
+      order: 3
+- uuid: 948746c7-237e-4e73-bc50-3b714ecb1ccf
+  type: SingleChoice
+  content: Bitcoin mining works on which of the principle?
+  hint: NoHint
+  explanation: Proof of Work consensus mechanism is used to mine Bitcoin.
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Proof of Authority
+      key: A
+      order: 0
+    - content: Proof of stake
+      key: B
+      order: 1
+    - content: Proof of work
+      key: C
+      order: 2
+    - content: Proof of History
+      key: D
+      order: 3
+- uuid: e9b6538b-a132-468e-8728-0818b9b3acba
+  type: SingleChoice
+  content: Bitcoin has got unlimited supply.
+  hint: NoHint
+  explanation: Bitcoin works on Proof of work mechanism.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: True because it can be mined easily using a mathematical algorithm
+      key: A
+      order: 0
+    - content: False because returns of solving mathematical algorithms are decreasing
+      key: B
+      order: 1
+    - content: True because miners mine Bitcoin every 10 minutes.
+      key: C
+      order: 2
+    - content: False because bitcoin is a volatile asset.
+      key: D
+      order: 3
+- uuid: 46a8b293-9028-4240-adf7-5a31b39f5415
+  type: SingleChoice
+  content: Who is the Founder of Ethereum?
+  hint: NoHint
+  explanation: Vitalik Buterin is the founder of Ethereum.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Pavel Durov
+      key: A
+      order: 0
+    - content: Vitalik Buterin
+      key: B
+      order: 1
+    - content: Changpen Zhao
+      key: C
+      order: 2
+    - content: Brian Armstrong
+      key: D
+      order: 3
+- uuid: 6c7083e5-ae8f-407d-bdbd-14f8b5327d95
+  type: SingleChoice
+  content: Ethereum mining works on the principle of ......
```

Comment:
Ethereum mining works on the principle of ......

I think so far it was proof of work and now they are switching to proof of stake. So we need to reword the question

Reply:
Done